libtool/PKGBUILD

```diff
@@ -65,7 +76,7 @@ build() {
 
 check() {
   cd ${srcdir}/${pkgbase}-${pkgver}
-  make check
+  # make check
```

Comment:
This commenting is not required. One can run check function in their system.

Reply:
The check sometimes got stuck for me for some reason. (But that already happened with 2.4.6 iirc.) So, I commented it out. I'll revert that.

Reply:
You can write it something like this `make check || warning "Tests failed"`. Or you can disable check() while building a package with `makepkg-mingw --nocheck` command.

Reply:
The issue wasn't that the checks were failing. But they were stuck for a very long time (maybe indefinitely). Maybe using `timeout 15m make check || warning "Tests failed"` would also have helped...

Reply:
Opinions may vary. Feel free to change that command as you like. Just I would suggest not to disable/comment out it.

Reply:
Done.